api/proto/teleport/accesslist/v1/accesslist_service.proto

```diff
@@ -0,0 +1,72 @@
+// Copyright 2023 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package teleport.accesslist.v1;
+
+import "google/protobuf/empty.proto";
+import "teleport/accesslist/v1/accesslist.proto";
+
+option go_package = "github.com/gravitational/teleport/api/gen/proto/go/teleport/accesslist/v1;accesslistv1";
+
+// AccessLisetService provides CRUD methods for Access List resources.
+service AccessListService {
+  // GetAccessLists returns a list of all access lists.
+  rpc GetAccessLists(GetAccessListsRequest) returns (GetAccessListsResponse);
+  // GetAccessList returns the specified access list resource.
+  rpc GetAccessList(GetAccessListRequest) returns (GetAccessListResponse);
+  // UpsertAccessList creates or updates an access list resource.
+  rpc UpsertAccessList(UpsertAccessListRequest) returns (google.protobuf.Empty);
+  // DeleteAccessList removes the specified access list resource.
+  rpc DeleteAccessList(DeleteAccessListRequest) returns (google.protobuf.Empty);
+  // DeleteAllAccessLists removes all access lists.
+  rpc DeleteAllAccessLists(DeleteAllAccessListsRequest) returns (google.protobuf.Empty);
+}
+
+// GetAccessListsRequest is the request for getting all access lists.
+message GetAccessListsRequest {}
```

Comment:
IMO we should add `ListObjects` for pagination, `GetObjects` for retrieving everything -- this seems to reflect our convention.

Reply:
I'll reiterate the advice to always go with pagination, but your call.

Reply:
I'm going to leave off pagination for now because my pagination functions for `UserGroups`, `SAMLIdPServiceProvider`, `OktaAssignment`, and `OktaImportRule` all pretty much went unused/only used as unpaginated bits (even where pagination applies, interestingly), which just ended up making listing requiring more boilerplate.